### DISCUSSION TITLE
activity overview issue

### DISCUSSION THREAD

**Discussion by thanasxda** [community/community#37938] [Other Feature Feedback, Questions, & Ideas] [answered]
i have unticked all activity overview on my profile and even deleted and repushed a repo because my activity still shows up. how can this be resolved? even this particular post shows up in my activity overview.
in other words this doesnt work for me:
https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/managing-contribution-settings-on-your-profile/showing-an-overview-of-your-activity-on-your-profile

**Comment by radar-blip** [comment, marked as answer]
Maybe this is what you're looking for (make profile private):  >> in settings

![profiles](https://user-images.githubusercontent.com/39196767/199142975-5fb269fc-b3a0-4f58-be2b-a7fe87e01dd6.png)

**Reply by DSDmark** [reply]
Click on that link and scroll down, then you can see these **contributions & Activity** section:-

[**contributions & Activity**](https://github.com/settings/profile)

**Reply by thanasxda** [reply]
i wonder how i could have overlooked that. thanks! it did hide my pinned repositories however but doesnt matter. did the job. :)

**Reply by DSDmark** [reply]
Hey @thanasxda,

I hope you got your answer, if you got. Then, you need to close this question by click **accept answer** button

**Reply by thanasxda** [reply]
bear with me im still a noob. thanks again

**Reply by radar-blip** [reply]
I see it worked for you. You're welcome. @DSDmark thanks to you too!
